### ISSUE TITLE
Conflict with Pehkui: Scaled-up player accelerates when sliding against a wall 

### ISSUE THREAD

**Issue by SIGSTACKFAULT** [ValkyrienSkies/Valkyrien-Skies-2#1043]
### Mod Name

Pehkui

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [x] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Fabric

### Issue description

With Pehkui and Valkyrien Skies, a curious physics bug emerges.

sliding against a wall while holding a directional key and space causes the player to accelerate upwards.

This might actually be a bug in Pehkui.

Recorded a video for you. I would embed it in GitHub but it doesn't like it. https://youtu.be/HIQsreidUMI

### Issue reproduction

Export from ATLauncher: [Pehkui & Valk conflic test 1.20.1.zip](https://github.com/user-attachments/files/18268360/Pehkui.Valk.conflic.test.1.20.1.zip)

### Logs

[latest.log](https://github.com/user-attachments/files/18268361/latest.log)

**Comment by SIGSTACKFAULT** [comment]
Also reported as https://github.com/Virtuoel/Pehkui/issues/570

**Comment by thetheaplant** [comment]
We've heard of similar things happening, which may or may not have been fixed in #927. Were you scaled when this bug occurred?

**Comment by SIGSTACKFAULT** [comment]
@thetheaplant it's only when scaled. it's minorly noticeable at 1.1x, but goes really crazy at ~1.3x. the link to a modpack i included should:tm: include a world with some commandblocks with the relevant commands so you can test it yourself.

**Comment by PriestOfFerns** [comment]
like thea said, https://github.com/ValkyrienSkies/Valkyrien-Skies-2/pull/927 should have fixed it, but I don't think the fix is in any public builds

**Comment by SIGSTACKFAULT** [comment]
Yep that sounds like the bug i'm experiencing. Can i get a vague ETA on that shipping (next week, next month, etc)?

**Comment by PriestOfFerns** [comment]
@SIGSTACKFAULT valkyrien skies isn't getting any released beta builds until VS2.5, which is somewhere between developing and bug fixing. 

So probably not soon

**Comment by SIGSTACKFAULT** [comment]
Would it be possible to backport #927 to `2.3.0-beta.5`? just for myself

**Comment by PriestOfFerns** [comment]
@SIGSTACKFAULT it is, though you might have to solve a merge conflict or add the code in manually. 

Although the code could also just not work on backport, due to api changes

**Comment by SIGSTACKFAULT** [comment]
(I'm out of town for the holidays and intellij was still downloading dependencies when I had to leave.)

**Comment by SIGSTACKFAULT** [comment]
[valkyrienskies-120-2.3.0-beta.5+e4f614ff82.jar.zip](https://github.com/user-attachments/files/18308273/valkyrienskies-120-2.3.0-beta.5%2Be4f614ff82.jar.zip)

found the compile button. here it is for any future internet travellers, if you're willing to trust a .jar uploaded by a stranger (which you shouldn't, really).
